config/config-example.js

```diff
@@ -645,6 +628,27 @@ exports.grouplist = [
 		modchat: true,
 		hiderank: true,
 	},
+	// Bots are ranked below Driver so that they can be kept out of modjoin % rooms.
+	// This was motivated by rhe desire to promote Global Bots whose owners do not hav Staff room access.
+	{
+		symbol: '*',
+		id: "bot",
+		name: "Bot",
+		inherit: '%',
+		jurisdiction: 'u',
+
+		addhtml: true,
+		tournaments: true,
```

Comment:
Yeah, pretty sure it's the `tournaments` permission

Reply:
Thanks!